src/strings.cpp

```diff
@@ -322,6 +322,19 @@ std::string GetString(StringID string)
 	return GetStringWithArgs(string, _global_string_params);
 }
 
+/**
+ * Resolve the given StringID and place into an existing std::string with all the associated
+ * DParam lookups and formatting.
+ * @param result The std::string to place the translated string.
+ * @param string The unique identifier of the translatable string.
+ */
+void GetStringInPlace(std::string &result, StringID string)
```

Comment:
Given the name and comment I would assume that this function also "clears" the string, but that's not what it's supposed to do looking at the uses.
Maybe something along the lines of `AppendStringInPlace`?